slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt

```diff
@@ -204,9 +204,28 @@ class SlideToActView @JvmOverloads constructor(
     private lateinit var mDrawableArrow: VectorDrawableCompat
 
     /** Tick drawable, is actually an AnimatedVectorDrawable */
-    private val mDrawableTick: Drawable
+    private var mDrawableTick: Drawable
     private var mFlagDrawTick: Boolean = false
 
+    var drawableTick: Int = 0
+        set(value) {
+            field = value
+            if (field != 0) {
+                mDrawableTick = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
This logic should be extracted inside a function and not duplicated in two places.

Reply:
done